docs/src/examples/components/Icon/Variations/IconExampleColor.shorthand.tsx

```diff
@@ -53,6 +44,18 @@ const IconExampleColor = () => (
         variables={{ color: 'cornflowerblue', borderColor: 'orangered' }}
       />
     </div>
+    <Text weight="bold">
+      USING THE <code>color</code> PROP:
+    </Text>
+    <div>
+      <ProviderConsumer
+        render={({ siteVariables: { emphasisColors, naturalColors } }) =>
+          _.take(_.keys({ ...emphasisColors, ...naturalColors }), 3).map(color => (
```

Comment:
this expression is very confusing - there should be a way to simplify this example. Actually, why slicing is necessary here?

Reply:
![screenshot 2018-12-19 at 19 38 04](https://user-images.githubusercontent.com/5442794/50290815-8759ba00-0475-11e9-8017-6439cab5aef5.png)
just to make the example fit nicely, any suggestions for improving, @kuzhelov ?

Reply:
@kuzhelov can I resolve this?